gestion_base_adresse/install/sql/upgrade/upgrade_to_0.3.0.sql

```diff
@@ -191,11 +191,11 @@ BEGIN
             OR NEW.suffixe IS DISTINCT FROM OLD.suffixe
             OR NEW.id_voie IS DISTINCT FROM OLD.id_voie
             OR ST_DISTANCE(NEW.geom, OLD.geom) > 0.0
-            OR (NEW.a_valider IS DISTINCT FROM OLD.a_valider AND OLD.a_valider) THEN
+            OR (NEW.valide IS DISTINCT FROM OLD.valide AND OLD.valide) THEN
```

Comment:
```suggestion
            OR (NEW.valide IS DISTINCT FROM OLD.valide AND NEW.valide) THEN
```
Il faut inverser la logique